src/Cache/Engine/RedisEngine.php

```diff
@@ -116,6 +117,8 @@ protected function _connect(): bool
                 );
             }
         } catch (RedisException $e) {
+            Log::error('RedisEngine RedisException: ' . $e->getMessage());
```

Comment:
This needs to be wrapped in a `class_exists(Log::class)` check other wise we will have to add `cakephp/log` as a dependency of the `cakephp/cache` package.

Reply:
Thanks. Should this use the FQN?

```php
            if (class_exists(\Cake\Log\Log::class)) {
                \Cake\Log\Log::error('RedisEngine RedisException: ' . $e->getMessage());
            }
```

Or is importing the class fine?

Reply:
You can `use` the class normally as long as it's wrapped in class_exists since it might fail to load.